lib/mimdb_web/controllers/genre_controller.ex

```diff
@@ -0,0 +1,62 @@
+defmodule MimdbWeb.GenreController do
+  use MimdbWeb, :controller
+
+  alias Mimdb.Movies
+  alias Mimdb.Movies.Genre
+
+  def index(conn, _params) do
+    genres = Movies.list_genres()
+    render(conn, "index.html", genres: genres)
+  end
+
+  def new(conn, _params) do
+    changeset = Movies.change_genre(%Genre{})
+    render(conn, "new.html", changeset: changeset)
+  end
+
+  def create(conn, %{"genre" => genre_params}) do
+    case Movies.create_genre(genre_params) do
+      {:ok, _genre} ->
+        conn
+        |> put_flash(:info, "Genre created successfully.")
+        |> redirect(to: Routes.genre_path(conn, :index))
+
+      {:error, %Ecto.Changeset{} = changeset} ->
+        render(conn, "new.html", changeset: changeset)
+    end
+  end
+
+  def show(conn, %{"id" => id}) do
```

Comment:
It would be ok to remove this method (and all show related code) if you don't think it will be used. Your code has removed the need for it...so it is safe to remove unless there are mitigating circumstances.

Reply:
Deleted show() method